Sources/StreamVideo/Utils/Store/StoreNamespace.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol StoreNamespace {
```

Comment:
This setup will allow us not only to instantiate stores easily (and configure them) but also to mock them for unit tests (as we can replace the executor with a new implementation and spy every execution).